client/css/widgets/pile.css

```diff
@@ -20,19 +29,19 @@
 }
 
 .pile .handle.small {
-  width: 20px;
-  height: 20px;
-  font-size: 10px;
+  width: calc(var(--phWidth)/2);
+  height: calc(var(--phHeight)/2);
+  font-size: calc(var(--phFontSize)/2);
```

Comment:
I'd prefer it if these were separate variables defaulted to these values. This gives the user more control over the size of the small handle. People with poor eyesight have trouble seeing the small handles :)

Reply:
hum you could just set you handle size to be 80. this is only triggered for cards that's are under 50 so you would know when to do that change.
also It would solve your problem directly.

this class is only used to avoid having to change the pile size manually if you set a card size to be almost the same size of the handle.

I'm still open to changes just point out why I don't think this is a solution to your problem

Reply:
OK, that makes sense. If a card is shrunk (using `enlarge`), what will happen to the size of the handle?

Reply:
neither scale or enlarge seen to have any effect on this.

Don't forget to solve the request if you agree with me.